hack/test-stirunimage.sh

```diff
@@ -14,6 +14,10 @@ function time_now()
 
 mkdir -p /tmp/sti
 WORK_DIR=$(mktemp -d /tmp/sti/test-work.XXXX)
+S2I_WORK_DIR=${WORK_DIR}
+if [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
This assumes Windows users use `cygwin`, is that a fair assumption? Hmm well, this is a bash script... already not something very Windows-friendly...

What happens to folks using `cmd.exe` or PowerShell?